classes/local/attributemapping.php

```diff
@@ -116,6 +120,10 @@ private function validate_field($table, $field) {
      * @throws InvalidArgumentException If the ID is invalid
      */
     private function validate_id($table, $id) {
+        if (!$id) {
+            return;
+        }
```

Comment:
Allowing non-selected values as users could leave the field blank and save the form.

Reply:
This should be reverted once multiple selection is implemented.